src/main/webapp/assets/modules/common/challenge/joinInternalChallenge.js

```diff
@@ -50,7 +56,7 @@ techlooper.directive("joinInternalChallenge", function (apiService, $translate,
           //  scope.challenge.$failedJoin = true;
           //}
           joiningRegistrant.reason == "UNMATCH_PASSCODE" && scope.joinInternalForm.passcode.$setValidity("matchPasscode", false);
-          joiningRegistrant.reason == "SINGLE_ACCOUNT" && scope.joinInternalForm.passcode.$setValidity("singleAccount", false);
+          joiningRegistrant.reason == "SINGLE_ACCOUNT" && scope.joinInternalForm.email.$setValidity("singleAccount", false);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.